app/src/main/java/com/sun/ise/ui/main/ViewPagerAdapter.kt

```diff
@@ -0,0 +1,17 @@
+package com.sun.ise.ui.main
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+
+class ViewPagerAdapter(fragmentManager: FragmentManager) : FragmentPagerAdapter(fragmentManager) {
+    private val mFragmentList = ArrayList<Fragment>()
```

Comment:
```suggestion
    private val fragments = ArrayList<Fragment>()
```
ở kotlin e ko cần khai báo m